### DISCUSSION TITLE
The detail transfer method of one NDArray LIST between machines.

### DISCUSSION THREAD

**Discussion by ChrisQiqiang** [apache/incubator-mxnet#16958] [Q&A]
# A question about kvstore.push
The tensors are transferred by calling the function named KVstore.push and pull. And it also supports the way like this:
Kvstore.Push(key_list, [ndarray1, ndarray2...] ,priority)
I cannot get the detail process, I wanna know whether the Ndarray list would be compressed into a file, then be transferred in a TCP/IP process.
Through my test, it may just a list and the Ndarrays in a list are transferred separately. I want to confirm this.

**Comment by samskalicky** [comment]
@mxnet-label-bot update [Question, KVStore]
